src/App.tsx

```diff
@@ -1,14 +1,51 @@
 /* eslint-disable max-len */
-import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { useEffect, useState } from 'react';
```

Comment:
These imports should come first, structure is always the same:

third party libraries, like react

local folders